src/controllers/DynamicFacetRequestBuilder.ts

```diff
@@ -31,3 +32,8 @@ export class DynamicFacetRequestBuilder {
     return this.request.filterFacetCount;
   }
 }
+
+export function determineFilterFacetCount(options: { filterFacetCount?: boolean }) {
+  const { filterFacetCount } = options;
+  return isUndefined(filterFacetCount) ? true : filterFacetCount;
+}
```

Comment:
Here is the logic used for `facet search` requests. See the function on line 27 for the logic used for `search` requests.

The complexity comes from getting access to `query.filterField`.
I opted to simplify by setting `filterFacetCount` to `true` by default if not specified ...

Reply:
I wish I remembered, but I don't, perhaps it should've been the other way around?

Reply:
The order is indeed flipped, confirmed with Daniel Lavoie.